test/fixtures.js

```diff
@@ -25,4 +25,22 @@ async function getSignersAndDeployContracts() {
 	return { admin, payer, payee1, payee2, streamManager, mockUSDT, maliciousToken };
 }
 
-module.exports = { getSignersAndDeployContracts }
\ No newline at end of file
+// Creating the stream
+async function createdOpenStream(payee) {
+  const { payer, streamManager, mockUSDT } = await loadFixture(getSignersAndDeployContracts)
+
+  const rate = 1500
+  const terminationPeriod = 18 * 24 * 3600; // 18 days
+  const cliffPeriod = 24 * 3600; // 24 hrs
+
+  // Creating stream
+  await streamManager.connect(payer).createOpenStream(
+    payee,
+    mockUSDT.address,
+    rate,
+    terminationPeriod,
+    cliffPeriod
+  );
+}
```

Comment:
```suggestion
async function createdOpenStream(payee) {
  const { payer, streamManager, mockUSDT, ...rest } = await loadFixture(getSignersAndDeployContracts)

  const rate = 1500
  const terminationPeriod = 18 * 24 * 3600; // 18 days
  const cliffPeriod = 24 * 3600; // 24 hrs

  // Creating stream
  await streamManager.connect(payer).createOpenStream(
    payee,
    mockUSDT.address,
    rate,
    terminationPeriod,
    cliffPeriod
  );
  /// here get streamId from events
}

return { payer, streamManager, mockUSDT, streamId, ...rest}
```

In the test file, just use this fixture instead of using fixture `getSignersAndDeployContracts` plus calling the function `createdOpenStream`

Reply:
@georgeciubotaru 
I don't understand what it gives (about events)

Reply:
![image](https://github.com/holdex/payment-streaming/assets/83241890/14edd44e-4ad6-43cf-9b3b-d3d07e7ebc22)

Reply:
@CTAPCKPIM Okay, then skip streamId

Reply:
@georgeciubotaru
I'm a little confused, what should I do?
Just add `return {....}` to `createdOpenStream`?

Reply:
yes, define that function as a fixture so you can use it directly

Reply:
@georgeciubotaru 
So that I do not use two textures in some tests, but use one (`createdOpenStream`)?

Reply:
@CTAPCKPIM Yes

Reply:
@CTAPCKPIM It makes sense because you don't have the value here.

you can try to define after the imports

const createStreamOwner = () => createOpenStream(1)
const createStreamAsOtherUser = () => createOpenStream(0)

and inside the fixture

to that

if (param === 0) {
	payee = payee0
} else {
	payee = payee1
}

Reply:
and use like that loadFixture(createStreamOwner) or `loadFixture(createStreamAsOtherUser)`